_posts/2022-04-01-First-steps-with-swift-concurrency.md

```diff
@@ -19,17 +19,17 @@ The application is based on the NASA API which allows for displaying the APOD (A
 
 Please find the github repo under this [link](https://github.com/swiftingio/WWDC21).
 
-In the below sections we will give you also some more context for our code. So the examples below will relay on our app. But no worries, those examples does not require you to know the whole codebase :)
+In the below sections we will give you also some more context for our code. So the examples below will rely on our app. But no worries, those examples does not require you to know the whole codebase :)
 
 
 # Fetch data from the network using async/await.
 
-The new async/await syntax is really straightforward. All you have to do is use a new API call on `URLSession` which is `let (data, response) = try await URLSession.shared.data(from: url)`. The await keyword is how you notice that a block of code doesn’t execute as one transaction. The function may suspend, and other things may happen while it’s suspended between the lines of the function. More than that, the function may resume onto an entirely different thread.
+The new async/await syntax is really straightforward. All you have to do is use a new API call on `URLSession` which is `let (data, response) = try await URLSession.shared.data(from: url)`. The await keyword is how you notice that a block of code doesn’t get executed as one transaction. The function may suspend, and other things may happen while it’s suspended between the lines of the function. More than that, the function may resume onto an entirely different thread.
 
 The full function which fetches the APOD data in the Networking layer in our case looks like this:
 
 ```swift
-public func fetchApods(startDate: Date, endDate: Date) async throws -> [ApodModel] {
+public func fetchApody(startDate: Date, endDate: Date) async throws -> [ApodModel] {
```

Comment:
Are you sure the replace was correct? I thought that `fetchApods` is the correct name 😉